scim-core/src/main/java/org/apache/directory/scim/core/repository/DefaultPatchHandler.java

```diff
@@ -426,5 +447,48 @@ public <T extends ScimResource> void applyMultiValue(T source, Map<String, Objec
         }
       }
     }
+
+    /**
+     *  Detects Azure Quirk mode.
+     *  Azure uses an out of spec patch operation that does not use an express,
+     *  but instead uses a remove with a value.  Detect this and convert it to an expression
+     *  <pre><code>
+     *   {
+     *     "op":"remove",
+     *     "path":"members",
+     *     "value":[{
+     *         "value":"<id>"
+     *     }]
```

Comment:
Note that this is how Azure structures the request, but in theory, it could have other key:value pairs here, e.g.
```
"value":[{
    "value":"<id>"
    "display":"someDisplayName",
    "ref":"https://scim-example.com/api/v2/Users/<id>",
}]
```